client/components/isolated-editor/client/components/block-editor-toolbar/more-menu/index.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { DropdownMenu } from '@wordpress/components';
+import { moreVertical } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+
+import WritingMenu from './writing-menu';
+import EditorMenu from './editor-menu';
+import LinkMenu from './link-menu';
+
+/** @typedef {import('../../../index').BlockEditorSettings} BlockEditorSettings */
+
+/**
+ * Close dropdown callback
+ * @callback OnClose
+ */
+
+/**
+ * More menu render callback
+ * @callback OnMore
+ * @param {BlockEditorSettings} settings - Settings
+ * @param {OnClose} onClose - Callback to close the menu
+ */
+
+const POPOVER_PROPS = {
+	className: 'edit-post-more-menu__content',
+	position: 'bottom left',
+};
+const TOGGLE_PROPS = {
+	tooltipPosition: 'bottom',
+};
+
+/**
+ * More menu
+ *
+ * @param {object} props - Component props
+ * @param {BlockEditorSettings} props.settings - Settings
+ * @param {OnClose} props.onClick
+ * @param {OnMore} props.renderMoreMenu
+ */
+const MoreMenu = ( { settings, onClick, renderMoreMenu } ) => (
+	<DropdownMenu
+		className="edit-post-more-menu"
+		icon={ moreVertical }
+		label={ __( 'More tools & options' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **83** times:
`translate( 'Hide more tools & options' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/isolated-block-editor)